src/rosconsole/rosconsole.cpp

```diff
@@ -362,8 +362,6 @@ void Formatter::print(void* logger_handle, ::ros::console::Level level, const ch
     color = COLOR_GREEN;
   }
 
-  ROS_ASSERT(color != NULL);
-
```

Comment:
The changes in this file seems to be unrelated too. Please revert and consider to create a separate PR for it.

Reply:
I came across this while fixing it for the upstream version to still print something.
But it is now in a separate PR: #34